modules/addDate.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function displayDateTime() {
+  const time = DateTime.now().toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  const printDate = document.querySelector('.date');
+  printDate.innerHTML = time; // Tue, 12 May 2020 23:50:21 GMT
+}
```

Comment:
- [ ] Please, it's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions, Kindly make sure to address the requested changes :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)